include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -87,31 +92,23 @@ class CMAES
    * @tparam CallbackTypes Types of callback functions.
    * @param function Function to optimize.
    * @param iterate Starting point (will be modified).
+   * @param stepSize Starting sigma/step size (will be modified).
    * @param callbacks Callback functions.
    * @return Objective value of the final point.
    */
   template<typename SeparableFunctionType,
-           typename MatType,
-           typename... CallbackTypes>
-  typename MatType::elem_type Optimize(SeparableFunctionType& function,
-                                       MatType& iterate,
-                                       CallbackTypes&&... callbacks);
+      typename MatType,
+      typename... CallbackTypes>
+      typename MatType::elem_type Optimize(SeparableFunctionType& function,
+          MatType& iterate,
+          double stepSize = 0.6,
```

Comment:
Parameters like the step size are usually parameters to the constructor itself---do you want to move that there, then add `StepSize()` functions to access and change it?

Reply:
Also, might be worth using a step size of `0.0` (the default) for "automatic", which can use the existing behavior of `0.3 * (upperBound - lowerBound)`.  (I guess you'd have to create some "default step size" function for each `TransformationPolicyType`.)

Reply:
Alright! I will add the initial step size as a parameter of the constructor instead. I also agree that adding a default step size method for each TransformationPolicyType would work.